Library/Format.swift

```diff
@@ -3,6 +3,9 @@ import Foundation
 import KsApi
 import Prelude
 
+// swiftlint:disable:next force_unwrapping
+public let UTCTimeZone = TimeZone(secondsFromGMT: 0)!
```

Comment:
I wonder if the estimated delivery date, on the data struct, should be modeled differently and hold onto `DateComponents` instead? A bit more work to make that work (with new formatting functions), but should make it easier to avoid these kinds of issues at compile time.